tests/server/test_models.py

```diff
@@ -0,0 +1,296 @@
+"""Test cases for server models."""
+
+import pytest
+from sqlmodel import Field, Session, SQLModel, create_engine
+from sqlmodel.pool import StaticPool
+
+from lilypad.server.models import (
+    ActiveVersionPublic,
+    BaseSQLModel,
+    FunctionCreate,
+    FunctionPublic,
+    FunctionTable,
+    ProjectCreate,
+    ProjectPublic,
+    ProjectTable,
+    PromptCreate,
+    PromptPublic,
+    PromptTable,
+    Provider,
+    ResponseFormat,
+    Scope,
+    SpanCreate,
+    SpanPublic,
+    SpanTable,
+    VersionCreate,
+    VersionPublic,
+    VersionTable,
+)
+
+
+@pytest.fixture
+def fixture_engine():
```

Comment:
let's actually change this back to just `engine`

Reply:
The convention should be:
1. Use `mock_` when the fixture actually mocks something (e.g. fake data, MagicMock, etc.)
2. Use clear naming for anything that is not a mock (e.g. `sqlite_engine` in this case)